provisioning_templates/provision/kickstart_default.erb

```diff
@@ -149,10 +149,10 @@ skipx
 
   # VLAN (only on physical is recognized)
   if iface.virtual? && iface.tag.present? && iface.attached_to.present?
-    if rhel_compatible && os_major >= 6
-      network_options.push("--vlanid=#{iface.tag}")
-    elsif rhel_compatible && os_major >= 7
+    if rhel_compatible && os_major >= 7
       network_options.push("--interfacename=vlan#{iface.tag}")
+    elsif rhel_compatible && os_major <= 6
+      network_options.push("--vlanid=#{iface.tag}")
```

Comment:
Previously we added this option in all cases, not just rhel_compatible. should we still do that?

Reply:
True, let me rethink this.